regression/witnesses/examples/example_yaml1.c

```diff
@@ -0,0 +1,25 @@
+void reach_error(){}
```

Comment:
Can you also add regression tests for the following scenarios?

- Nested structures
- Floating-point values with precision
- Branching traces (multiple paths)
- Function calls/returns

Reply:
Thank you.